rewrite-java/src/main/java/org/openrewrite/java/ChangePackage.java

```diff
@@ -112,15 +114,48 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
                         }
                     }
                     stopAfterPreVisit();
```

Comment:
The stopAfterPreVisit() method should be called before the return statement, so that the tree visit is aborted.